### ISSUE TITLE
No longer compatible with `smile/elasticsuite` v2.10.4 (works with 2.10.3).

### ISSUE THREAD

**Issue by frqnck** [magesuite/elasticsuite-addons#1]
```
Fatal error: Declaration of MageSuite\ElasticSuiteAddons\Rule\Condition\Product\SpecialAttribute\IsNew::getSearchQuery(Smile\ElasticsuiteCatalogRule\Model\Rule\Condition\Product $condition) must be compatible with Smile\ElasticsuiteCatalogRule\Api\Rule\Condition\Product\SpecialAttributeInterface::getSearchQuery() in /app/vendor/creativestyle/magesuite-elasticsuite-addons/Rule/Condition/Product/SpecialAttribute/IsNew.php on line 16
```

**Comment by diwipl** [comment]
Thank you for reporting. 
We're also experiencing it. We're working on a fix.

**Comment by frqnck** [comment]
Thanks goes to you (and the rest of the team). Keep the great work.

**Comment by tim-breitenstein-it** [comment]
Same error for me!

**Comment by tim-breitenstein-it** [comment]
Solved for me with: [v1.2.2](https://github.com/magesuite/elasticsuite-addons/releases/tag/v1.2.2)

**Comment by diwipl** [comment]
I can confirm that we have fixed above issue within v1.2.2 release.
